google-cloud-spanner/src/test/java/com/google/cloud/spanner/SpannerOptionsTest.java

```diff
@@ -518,6 +519,14 @@ public void testSetClientLibToken() {
             .build();
     assertThat(options.getClientLibToken()).isEqualTo(hibernateToken);
 
+    options =
+        SpannerOptions.newBuilder()
+            .setProjectId("some-project")
+            .setCredentials(NoCredentials.getInstance())
+            .setClientLibToken(pgAdapterToken)
+            .build();
+    assertThat(options.getClientLibToken()).isEqualTo(pgAdapterToken);
```

Comment:
nit: we prefer the use of `assertEquals(expected, actual)` (add `import static org.junit.Assert.assertEquals` at the top of the file to import it)

I know that it is not consistent with the other tests here, but the general rule of thumb is to use this method for new code, even when it is different from the surrounding code.

Reply:
Thanks @olavloite for mentioning this. I've changed it and will remember this in future